include/oomph/communicator.hpp

```diff
@@ -220,20 +220,23 @@ class communicator
     }
 
     template<typename T>
-    [[nodiscard]] send_request send_multi(message_buffer<T> const& msg,
+    [[nodiscard]] send_request_vector send_multi(
```

Comment:
```suggestion
    send_request_vector send_multi(
```
maybe we can remove the nodiscard attribute for all the sends/recvs?